packages/blitz-auth/src/client/index.tsx

```diff
@@ -274,9 +281,11 @@ function withBlitzAuthPlugin<TProps = any>(Page: ComponentType<TProps> | BlitzPa
               : formatWithValidation(redirectAuthenticatedTo)
 
           debug("[BlitzAuthInnerRoot] redirecting to", redirectUrl)
```

Comment:
move this log into the if statement (and might as well add the other url code in there also)